lib/report_portal/cucumber/formatter.rb

```diff
@@ -7,12 +7,10 @@ module Cucumber
     class Formatter
       # @api private
       def initialize(config)
-        ENV['REPORT_PORTAL_USED'] = 'true'
```

Comment:
The intention of this environment variable was to ease checking if the Cucumber process was started with a Report Portal formatter or not. Do you propose to use some mechanism to check it?

Reply:
who is the use for this? I can get all the formatters options before cucumber starts in an after configuration hook
```rb
 AfterConfiguration do |config|
  config.formats
end
```

Reply:
E.g. embed additional logs if tests are run against Report Portal and don't embed them otherwise

Reply:
that is hacking and should not be part of generic formatter, in fact, if you need this flag use after configuration hook to enable it.

Reply:
I see. Sounds good enough.